ruby-on-rails/ruby-on-rails-quiz.md

```diff
@@ -345,9 +345,9 @@ after_find :decrypt_data
 end
 ```
 
-- [ ] `MyModel.first.update(field: 'example')`
-- [x] `MyModel.where(id: 42)`
-- [ ] `MyModel.first.destroy`
+- [x] `MyModel.first.update(field: 'example')`
```

Comment:
It works even when destroying, because the important part is the method that comes before destroying.